docs/topics/rules-important-links.adoc

```diff
@@ -6,6 +6,6 @@
 [id="rules-important-links_{context}"]
 = Additional resources
 
-* {ProductShortName} Javadoc: http://windup.github.io/windup/docs/latest/javadoc
+// * {ProductShortName} Javadoc: http://windup.github.io/windup/docs/latest/javadoc
```

Comment:
Why have you comments this out on this line, but included in line 108

Reply:
@anarnold97 I have checked each of the published MTA documents and do not see "Javadoc" in any of them, except for this location. 

You can see it's not in the preview build here. 

![image](https://github.com/windup/windup-documentation/assets/60698649/2646c5e5-7c62-49ff-b8d0-ba9508c616e3)

Please let me know the file and line you see it in. Although I have checked, i must have missed it. 

Or did i misunderstand your comment?

Reply:
@RichardHoch - see [comment](https://github.com/windup/windup-documentation/pull/893/files#r1634510835) . You have replaced one reference to JavaDoc and commented one out